suzieq/poller/inventory/inventory_sources_base/inventory.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, add_task_fn: Callable, **kwargs) -> None:
         self.jump_host = kwargs.pop('jump_host', None)
         self.jump_host_key_file = kwargs.pop('jump_host_key_file', None)
         self.ignore_known_hosts = kwargs.pop('ignore_known_hosts', False)
-        self.user_password = kwargs.pop('password', None)
+        self.user_password = kwargs.pop('user_password', None)
```

Comment:
username and password are fairly well-known terms. Why rename it user_password? Just a q, doesn't affect the approval whichever way you proceed